Library/Models/SurveyPublicId.cs

```diff
@@ -21,6 +21,11 @@ namespace Nfield.Models
     public class SurveyPublicId
     {
         /// <summary>
+        /// The public id that is used to link to the survey
+        /// </summary>
+        public string Id { get; set; }
```

Comment:
I don't know why you would need this. The surveyId/linkType combination should be enough.